### DISCUSSION TITLE
How to keep only plain text in the act of pasting

### DISCUSSION THREAD

**Discussion by redpandaex** [ueberdosis/tiptap#2278] [General Questions]
In the pasting behavior, I need to remove the label that comes with the mention, and only keep the text. Using handlePaste, I can already get the plain text, but I want to know how to proceed.thanks

**Comment by philippkuehn** [comment]
Do you mean that you want to prevent copying mentions within the editor? You can update the pasted content content with these hooks:

https://prosemirror.net/docs/ref/#view.EditorProps.transformPastedHTML
https://prosemirror.net/docs/ref/#view.EditorProps.transformPasted

**Reply by redpandaex** [reply]
Thank you. I have tried to use these two hook functions and use regular expressions to remove html tags, but errors will inevitably be reported. For example: Position xxx out of range. Maybe it is because I used other plugins. .I am currently using the handlePaste function, but I don’t know how to change the pasted content. Maybe new Slice()?

**Reply by philippkuehn** [reply]
That’s why I sent you links to other hooks 😉

**Reply by redpandaex** [reply]
Okay, thank you very much for your reply! I will learn more about it again
